module/module.go

```diff
@@ -507,3 +503,24 @@ func (m *Module) AddModelFromRegistry(ctx context.Context, api resource.Subtype,
 func (m *Module) OperationManager() *operation.Manager {
 	return m.operations
 }
+
+// addConvertAttributesToConfig uses the MapAttributeConverter to fill in the
+// ConvertedAttributes field from the Attributes.
+func addConvertAttributesToConfig(cfg *config.Component) error {
```

Comment:
```suggestion
func convertAttributes(cfg *config.Component) error {
```

That's a really awkward name otherwise.

Reply:
If not that, then maybe `addConvertedAttributes`

Reply:
Agreed -- done